client/my-sites/comments/comment/comment-actions.jsx

```diff
@@ -56,7 +56,7 @@ export class CommentActions extends Component {
 			window.confirm( this.props.translate( 'Delete this comment permanently?' ) )
 		) {
 			this.props.deletePermanently();
-			this.props.redirect();
+			this.props.redirect?.();
```

Comment:
It is not required for a component to pass the `redirect` prop. This component appears to be used in a couple places, and at least one of those does not include the redirect prop. @tyxla added this a few months ago in #61067 to redirect from the comments screen in some cases. I'm thinking that in the other cases, the redirect might not be important so it isn't added.